ui/src/voxetta-frontend.js

```diff
@@ -0,0 +1,3 @@
+import { VoxettaFrontend } from './VoxettaFrontend.js';
+
+customElements.define('voxetta-frontend', VoxettaFrontend);
```

Comment:
is there a reason why the scaffold separates this definition into another file? can we not / should we not just append line 3 to VoxettaFrontend.js?

Reply:
FYI I made this change in the latest commit, comment as you will

Reply:
FYI internally we use the typescript decorator on the class definition

```javascript
@customElement('my-cool-ele')
export class MyCoolEle extends LitElement
```